crates/aptos-inspection-service/src/server/jeprof.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
```

Comment:
if possible, let's rewrite it in rust and put it in the aptos-profiler crater as well